torch/nn/modules/loss.py

```diff
@@ -1346,6 +1346,20 @@ class MultiMarginLoss(_WeightedLoss):
             elements in the output, ``'sum'``: the output will be summed. Note: :attr:`size_average`
             and :attr:`reduce` are in the process of being deprecated, and in the meantime,
             specifying either of those two args will override :attr:`reduction`. Default: ``'mean'``
+
+    Shape:
+        - Input: :math:`(N, C)` where `N` is the batch size and `C` is the number of classes.
```

Comment:
formatting nit: \`N\` -> :math:\`N\` and \`C\` -> :math:\`C\` makes this look a bit nicer

Reply:
Also, since the input allows for no batch dim, this should read:

```
Input: :math:`(N, C)` or :math:`(C)`, where :math:`N` is the batch size and :math:`C` is the number of classes.
```